src/ossl/ecc.rs

```diff
@@ -709,3 +716,149 @@ impl Verify for EccOperation {
         Ok(self.output_len)
     }
 }
+
+pub const NONE_NAME: &[u8; 1] = b"\0";
```

Comment:
This is the empty c string. Maybe we can find one in openssl headers though.

I initially though this is for both the NULL (plain)  and NIST, but looks like the NIST KDF does no go there and is not implemented in openssl at all so I will drop that.

https://github.com/openssl/openssl/blob/7afa7731e924d5ac10fc992d8cd777f407d33af9/providers/implementations/exchange/ecdh_exch.c#L259

If we would like to support this, I think we would have to implement it with the separate KDF calls then.

Reply:
This might even work without explicit setting of this to empty string if we want the plain ... might be nicer to exclude that, but initially I wanted to be as specific as possible in regards to the params.

Reply:
Removed the setting of empty string. I set only the x963 when needed now.